qiita_db/metadata_template/sample_template.py

```diff
@@ -172,11 +194,18 @@ def delete(cls, id_):
 
         conn_handler.add_to_queue(
             queue,
+            "DELETE FROM qiita.{0} where {1} = %s".format(cls._log_table,
+                                                          cls._id_column),
+            (id_,))
+
+        conn_handler.add_to_queue(
+            queue,
             "DELETE FROM qiita.{0} where {1} = %s".format(cls._column_table,
                                                           cls._id_column),
             (id_,))
 
         conn_handler.execute_queue(queue)
+        cls._update_analyses(id_)
```

Comment:
same here. A sample template cannot be removed if there is data using it. No need for this line.